Zone5/Activities/Activity.swift

```diff
@@ -6,23 +6,24 @@ public struct Activity: Searchable, Hashable {
 	public var id: Int
 
 	/// The type of activity this activity's `id` is related to.
-	public var activity: ActivityResultType?
-
-	/// The sport related to this activity.
-	public var type: ActivityType?
+	public var type: ActivityResultType?
+	
```

Comment:
Funny, I was just looking at this yesterday and noted that it was wrong. 

Yes, you are correct, the ActivityResultType field should be called type. There is also an ActivityType field which should be called sport. So both fields had incorrect names.

Can you please update the PR to add in
`public var sport: ActivityType?`

Thanks!

Reply:
Updated :+1: